docs/general/about.md

```diff
@@ -9,11 +9,19 @@ Jellyfin is a Free Software Media System that puts you in control of managing an
 
 Jellyfin seeks to continue development of the original Emby project with a Free Software ethos. It is committed to bringing all its users access to the best possible Media System, developed entirely by a community of volunteers who contribute code, documentation, translations, and support to the project.
 
-The Jellyfin project was started in early December 2018 as a result of Emby's decision to take their 3.6 release closed-source, as well as various philosophical differences with the core developers. Jellyfin seeks to be the free software alternative to Emby and Plex to provide media management and streaming from a dedicated server to end-user devices.
-
 You can find our main repository on [GitHub](https://github.com/jellyfin/jellyfin) as well as our [organization](https://github.com/jellyfin) page.
 
-## Core Team
+## Why did you fork?
+
+The Jellyfin project was started in early December 2018 primarily as a result of Emby's decision to take their next release (4.x) closed-source. Several members of the future Jellyfin team, including Joshua, Andrew, Vasily, and dkanada, were actively observing the Emby situation due to previously-reported GPL violations and a growing discontent with the Emby team for their practices and lack of respect for the FLOSS philosophy and ethos. These including the addition of a paywall for gratis users in an early 3.x release, the hiding/removal of client app and server code over several years, a lack of openness to community contributions, and various other reasons. Upon the announcement of the close-source Emby 4 release, this group quickly gathered to fork as many repositories as (legally) possible and begin the process of renaming the project Jellyfin.
+
+Since December 2018, the project has grown immensely, through several speedbumps and waves of activity, gaining contributors from around the world and of all skill levels and experiences, and we continue to advance the project as best as we can. We want to be the stweards of a truly FLOSS option in the video streaming space against Plex and Emby, and to remain a good proof of the power of Free and Open-Source software.
```

Comment:
```suggestion
Since December 2018, the project has grown immensely, through several speedbumps and waves of activity, gaining contributors from around the world and of all skill levels and experiences, and we continue to advance the project as best as we can. We want to be the stewards of a truly FLOSS option in the video streaming space against Plex and Emby, and to remain a good proof of the power of Free and Open-Source software.
```